packages/@aws-cdk/aws-appmesh/lib/virtual-gateway-listener.ts

```diff
@@ -51,6 +66,16 @@ export interface VirtualGatewayListenerConfig {
   readonly listener: CfnVirtualGateway.VirtualGatewayListenerProperty,
 }
 
+/**
+ * Renders the TLS config for a listener
+ */
+function renderTls(scope: cdk.Construct, tlsCertificate: TlsCertificate): CfnVirtualGateway.VirtualGatewayListenerTlsProperty {
```

Comment:
Please move this to the bottom of the file (next to the existing private functions in this file).